experimental/untwisted.py

```diff
@@ -82,29 +93,30 @@ def publish(self, text):
         for x in self.sessions:
             x.write(text)
 
+
 class JSFeed(Feed):
     def __init__(self, callback="callback"):
         Feed.__init__(self)
         self.callback = callback
 
     def publish(self, obj):
         web.debug("publishing")
-        Feed.publish(self,
-          '<script type="text/javascript">window.parent.%s(%s)</script>' % (self.callback, simplejson.dumps(obj) +
-          " " * 2048))
+        Feed.publish(
+            self,
+            '<script type="text/javascript">window.parent.%s(%s)</script>'
+            % (self.callback, simplejson.dumps(obj) + " " * 2048),
+        )
+
 
 if __name__ == "__main__":
     mfeed = JSFeed()
 
-    urls = (
-      '/', 'view',
-      '/js', 'js',
-      '/send', 'send'
-    )
+    urls = ("/", "view", "/js", "js", "/send", "send")
```

Comment:
This is not acceptable.

Reply:
Agree. We lose readability for this (urls) reformat.